src/processors/OpenAPIInputProcessor.ts

```diff
@@ -160,6 +172,27 @@ export class OpenAPIInputProcessor extends AbstractInputProcessor {
     }
   }
 
+  private iterateParameters(
+    parameters:
+      | (OpenAPIV3.ReferenceObject | OpenAPIV3.ParameterObject)[]
+      | undefined,
+    path: string,
+    inputModel: InputMetaModel,
+    options?: ProcessorOptions
+  ) {
```

Comment:
You dont want to introduce an option so the user can specify whether they want to include parameters or not? 🤔

Reply:
The parameters are just as much part of the OpenAPI spec as the requests and responses, so I don't see much of a use case in having them as an option. The extra cost is minimal. Happy to add one if you have strong views. I don't see anything similar on `ProcessorOptions` for other document formats.

Reply:
Thats because in for example AsyncAPI input processor, we only generate message payloads, not parameters, even though we probably should 😄 

But no I dont have any strong feelings for the OpenAPI input, so here it's fine 😄